BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -92,7 +93,7 @@ private void updateLocation() {
 
         // The root mean square of multilateration is used to filter out inaccurate locations.
         // Adjust value to allow location updates with higher deviation
-        if (multilateration.getRMS() < ROOT_MEAN_SQUARE_THRESHOLD_STRICT) {
+        if (multilateration.getRMS() < ROOT_MEAN_SQUARE_THRESHOLD) {
```

Comment:
This should be `rootMeanSquareThreshold `

Reply:
Sorry for that